src/pip/_internal/index.py

```diff
@@ -669,7 +669,7 @@ def _link_package_versions(self, link, search):
             version = egg_info_matches(egg_info, search.supplied, link)
         if version is None:
             self._log_skipped_link(
-                link, 'wrong project name (not %s)' % search.supplied)
+                link, 'Missing project version')
```

Comment:
I feel this should include search.supplied in a bracket? That way all the information currently shown would still be shown...

```py
            self._log_skipped_link( 
                link, 'Missing project version ({})'.format(search.supplied))
```

Reply:
I don't have a strong preference, but it is worth noting the rest of the file uses `%` formatting. I also tried to match the phrasing of the other log strings (e.g. [L665](https://github.com/pypa/pip/blob/master/src/pip/_internal/index.py#L665) and [L639](https://github.com/pypa/pip/blob/master/src/pip/_internal/index.py#L639)). Let me know if you'd prefer your example.